src/part-4-intro.md

```diff
@@ -1,7 +1,8 @@
-# Part 4: The Type System
+# Part 4: Analysis
 
 This part discusses the many analyses that the compiler uses to check various
-properties of the code and to inform later stages. This includes the
+properties of the code and to inform later stages. Typically, this is what people
+mean when they talk about "Rust's type system". This includes the
 representation, inference, and checking of types, the trait system, and the
 borrow checker. These analyses do not happen as one big pass or set of
```

Comment:
Is borrow checking considered part of the type system? That's surprising, given that type checking and borrow checking are separate functions.

Reply:
Lifetimes are part of the type of a definition, so I guess it is part of the "type system".